meta-ostro/conf/distro/include/ostro-os-core-x11.inc

```diff
@@ -0,0 +1,3 @@
+DISTRO_FEATURES_append = " x11"
+DISTRO_FEATURES_append = " opengl"
+IMAGE_INSTALL_append = " packagegroup-core-x11 xterm twm"
```

Comment:
Default xinitrc uses those. Without those you don't get anything at all included that would have sensible output on X11. Many people like to run some tests on the image using terminal.